js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js

```diff
@@ -171,6 +184,58 @@ define([
             CsvUtils.saveAsCsv(exprt);
         }
 
+        createNewSet(analysis) {
+        	this.loading(true);
+        	const conceptIds = this.extractConceptIds(analysis);
+        	vocabularyProvider.getConceptsById(conceptIds)
+        		.then(({ data: items }) => { this.initConceptSet(items) })
```

Comment:
Please use async / await

Reply:
updated

Reply:
Where is it "updated"? I see `.then`, `.catch` and `.finally`

Reply:
yes. fixed